src/main/java/org/cryptomator/common/networking/SSLContextProvider.java

```diff
@@ -0,0 +1,28 @@
+package org.cryptomator.common.networking;
+
+import org.cryptomator.integrations.common.IntegrationsLoader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.net.ssl.SSLContext;
+import java.security.SecureRandom;
+import java.util.ServiceLoader;
+import java.util.stream.Stream;
+
+public interface SSLContextProvider {
+
+	Logger LOG = LoggerFactory.getLogger(SSLContextProvider.class);
```

Comment:
unused, isn't it?

Reply:
Done in https://github.com/cryptomator/cryptomator/pull/3675/commits/66a0cfe7dd6d495887dc029ee63baaede65dc335